data/flashcards.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
I think the .gitignore needs to be modified s.t. data/* won't be version controlled. Previously it was working but now due to some refactoring issues it no longer ignores.

Reply:
should remove this file (user local storage)

Reply:
added to gitignore in new PR